src/main/java/servlets/MirrorServlet.java

```diff
@@ -0,0 +1,18 @@
+package servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class MirrorServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {

```

Comment:
try to use full names of the parameters

https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit